frontend/src/scenes/dashboard/DashboardItems.tsx

```diff
@@ -99,7 +99,7 @@ export function DashboardItems({ inSharedMode }: { inSharedMode: boolean }): JSX
                         }
                         updateItemColor={updateItemColor}
                         isDraggingRef={isDragging}
-                        inSharedMode={inSharedMode}
+                        dashboardMode={dashboardMode}
                         isOnEditMode={dashboardMode === DashboardMode.Edit}
```

Comment:
Now that we're passing the `dashboardMode` fully downstream, seems like we should refactor this too.

Reply:
:+1: Agreed, will open a follow-up